docker/dispatch.sh

```diff
@@ -7,9 +7,9 @@ _term() {
   exit "$status"
 }
 
-case "$(uname)" in
-	Darwin*)	HOST_MACHINE=Mac;;
-	*)		HOST_MACHINE=Linux;;
+case "$(docker info | grep -E '^\s*Name' | awk '{ print $2 }')" in
+	docker-desktop*) HOST_MACHINE=Mac;;
```

Comment:
I had this set up so that I could run docker from within containers. I guess we're just looking for the host os here, we're really looking to check if we're running in CI and whether we need to set uid/gid in the docker run line.

I'll back this out for now and see if there's a better way to identify docker on linux